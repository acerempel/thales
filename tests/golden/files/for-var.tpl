<ul>
  {for vegetable in vegetables}
  <li>I like {vegetable}</li>
  {end}
</ul>
