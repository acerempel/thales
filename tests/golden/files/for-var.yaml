vegetables:
  - potato
  - leek
  - onion
  - turnip
